SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -370,6 +370,31 @@ ESchemaComponentType USpatialClassInfoManager::GetCategoryByComponentId(Worker_C
 	return ESchemaComponentType::SCHEMA_Invalid;
 }
 
+const FRPCInfo& USpatialClassInfoManager::GetRPCInfo(UObject* Object, UFunction* Function)
+{
+	check(Object != nullptr && Function != nullptr);
+
+	const FClassInfo& Info = GetOrCreateClassInfoByObject(Object);
+	const FRPCInfo* RPCInfoPtr = Info.RPCInfoMap.Find(Function);
+
+	// We potentially have a parent function and need to find the child function.
+	// This exists as it's possible in blueprints to explicitly call the parent function.
+	if (RPCInfoPtr == nullptr)
+	{
+		for (auto It = Info.RPCInfoMap.CreateConstIterator(); It; ++It)
+		{
+			if (It.Key()->GetName() == Function->GetName())
+			{
+				// Matching child function found. Use this for the remote function call.
+				RPCInfoPtr = &It.Value();
+				break;
+			}
+		}
+	}
+	check(RPCInfoPtr != nullptr);
```

Comment:
When does this happen? This isn't a postcondition so I don't think we should be using check here.

Reply:
It should never happen.

Reply:
Then we should remove the check

Reply:
My understanding was that check is there to explicitly state it must always be valid.

Reply:
We generally use `check`s as sanity checks so that if a developer changes something that this code was relying on, they will run into a `check` and will have to resolve this logic.